symfony/plugins/orangehrmPerformanceTrackerPlugin/test/model/service/PerformanceTrackerServiceTest.php

```diff
@@ -17,6 +17,8 @@
  * Boston, MA  02110-1301, USA
  */
 
+use OrangeHRM\Performance\test\Service\PerformanceTrackerService;
```

Comment:
Add `"OrangeHRM\\Tests\\Performance\\": "./plugins/orangehrmPerformancePlugin/test"` to symfony/composer.json
```suggestion
use OrangeHRM\Tests\Performance\PerformanceTrackerService;
```

Reply:
Changed